### PR TITLE
 Add Icon on executable

### PR DESCRIPTION
**Pull request by Zangetsu38**
Add Icon on exe and on gui, this icon is temporary here, prepared file for later with real icon.
Preview
capture d ecran 2018-02-04 00 23 00

Move exe and dll file on bin folder like that, path changed depending you using Debug or Release same before, but just now is more easy for found program compiled.
capture d ecran 2018-02-04 05 15 38

### CODE REVIEW

**Comment by petmac** [comment]
Thanks for this. Would it be possible to split this pull request so the icon and version changes are separate?

**Comment by Zangetsu38** [comment]
Yes if you want, i have set like this because is small commit

**Comment by Zangetsu38** [comment]
So i have spleeted commit in two part

**Comment by Zangetsu38** [comment]
@petmac you can say me number corresping on Appveyor Vita3K ? 
It is on Setting, and badge, have number like that "https://ci.appveyor.com/api/projects/status/2lgwjex532xj4ndy?svg=true"

**Comment by Zangetsu38** [comment]
I needed that for set correct project for badge build

**Comment by Zangetsu38** [comment]
rebased and so removed Fix for correct link in readme

**Comment by Zangetsu38** [comment]
@petmac So i have spilted this in two PR and changed name commit

File: appveyor.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ build:
   
 after_build:
  - ps: | # Package artifact
-    7z a -m0=LZMA2 -mx9 $env:BUILD .\build-windows\src\emulator\Release\*
```
Comment (petmac) [inline appveyor.yml]: Is this OK? Do we need to do this to change the icon?

File: src/emulator/CMakeLists.txt [hunk 2]
```diff
@@ -34,12 +34,17 @@ load_self.h
...
+Vita3K.rc
 vpk.cpp
 vpk.h
 )
 
 target_link_libraries(emulator PRIVATE cpu elfio host modules nids vita-toolchain)
-set_target_properties(emulator PROPERTIES OUTPUT_NAME Vita3K)
+set_target_properties(emulator PROPERTIES OUTPUT_NAME Vita3K
+    ARCHIVE_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bin"
+    RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bin")
```
Comment (petmac) [inline src/emulator/CMakeLists.txt]: What happens if we don't change the output directory?

File: appveyor.yml (hunk 1 above)
Comment (Zangetsu38) [reply appveyor.yml]: no, it is for move executable on bin folder, same in my scree, obligatotory for compile fine on appveyor

File: src/emulator/CMakeLists.txt (hunk 2 above)
Comment (Zangetsu38) [reply src/emulator/CMakeLists.txt]: bad folder, compiled on "build-windows\src\emulator\(Release,Debug)" like that move on "build-windows\bin\(Release,Debug)"

Comment (petmac) [reply src/emulator/CMakeLists.txt]: OK. If it's clearer and works for everyone that's good. Thanks!

**Pull request merged**